include/ql/utils/json.h

```diff
@@ -17,7 +17,7 @@
 // check existence of JSON key within node, see PR #194
 #define QL_JSON_EXISTS(node, key)  ((node).count(key) > 0)
 
-#define QL_JSON_FATAL(s) QL_FATAL("Error in JSON definition: " << s)   // NB: FATAL prepends "Error : "
+#define QL_JSON_FATAL(s) QL_FATAL("Error in JSON definition: " << s)   // NB: FATAL prepends "Error : ". FIXME: replace by QL_JSON_ERROR
```

Comment:
I see many FIXMEs. Several, like this one, seem easy to implement.

Reply:
Solved: there was actually only one remaining reference to the old QL_JSON_FATAL, now QL_JSON_ERROR (from exception.h)  is used throughout